data/mods/CRT_EXPANSION/items/crt_gunmods.json

```diff
@@ -39,24 +39,13 @@
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "shot": 1 } } ]
   },
   {
-    "id": "bthk_stock",
+    "id": "folding_bthk_stock",
```

Comment:
Why not just make them all vanilla instead of leaving one in CRIT?

Reply:
CRIT had both versions, only one actually spawned though.
I couldn't find any case of folding butt hook stock, only permanent ones, so I left folding one in the mod, just added spawn and updated it with new values.